src/util/util_errors.h

```diff
@@ -203,6 +203,14 @@ enum sssd_errors {
 #define ENONET EHOSTDOWN
 #endif
 
+#ifndef ELIBACC
+#define ELIBACC EACCES
+#endif
+
+#ifndef ELIBBAD
+#define ELIBBAD EBADF
```

Comment:
`ELIBBAD` is handled internally:
```
providers/data_provider/dp_targets.c:290:    if (!target->explicitly_configured && (ret == ELIBBAD || ret == ENOTSUP)) {
```

I wonder if re-using `EBADF` can trigger this codepath unintentionally.

Reply:
We discussed this internally and would like to see a consistent approach for missing error codes, i.e. "assign new numbers".

Unfortunately, https://github.com/llvm/llvm-project/blob/main/libcxx/include/errno.h#L84 also doesn't define `ELIBACC`/`ELIBBAD`
Maybe just assign something past `define EMLINK 9979` (with a margin).

This shouldn't have any practical impact because it's only used within 'providers/data_provider/' - both set and read - so specific value isn't really important.

Reply:
Maybe it is then better to use `ELAST` as a starting point?

Reply:
I'd prefer this to be as generic as possible.
Linux, for example, doesn't have `ELAST` (I know this branch isn't going to be used on Linux at all, but just as an example).